src/utils/content-to-jsx.tsx

```diff
@@ -52,6 +52,12 @@ export const contentToJsx = (content: Content, index?: number): JSXContent => {
       return <li key={index}>{contentToJsx(content.content)}</li>;
     case 'List':
       return content.order === 'Ascending' ? <ol key={index}>{contentToJsx(content.items)}</ol> : <ul key={index}>{contentToJsx(content.items)}</ul>;
+    case 'Claim':
+      return <section key={index}>
+        {content.title ? <h3>{contentToJsx(content.title)}</h3> : ''}
+        <h4>{content.label}</h4>
+        {contentToJsx(content.content)}
```

Comment:
Suggest that either both label and title should be in the same header (this would be my preferred rendering), or that label should appear before title.

Reply:
Yup, thanks. That LGTM 👍